common/defaults/project/catalog-repositories/dkp-catalog-applications.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: GitRepository
+metadata:
+  name: dkp-catalog-applications
+  namespace: ${releaseNamespace}
+  labels:
+    kommander.d2iq.io/gitrepository-type: catalog
+    kommander.d2iq.io/gitapps-gitrepository-type: dkp
+spec:
+  gitImplementation: go-git
+  interval: 1m0s
+  ref:
+    branch: main
```

Comment:
💭 are we always going to point to main? I thought we initially wanted to tag releases and try to match kommander releases. I’m not sure how we could substitute this for the release version though.

Reply:
Do we want to use a substitution variable here and set the version in kommander? (we would use the same version as kommander I guess, if we are keeping those versions coupled)

Reply:
Discussed this offline, decided to go with `branch: main` for now and as part of release process, we will swap this to `tag: v2.2.0`. I ll create a PR to reflect this at https://github.com/mesosphere/kommander/blob/main/CONTRIBUTING.md#publishing-a-new-release once this PR is merged.

Reply:
Update the KEP and will update the release notes to change https://github.com/mesosphere/kommander-cli/pull/353/files#diff-2c390594da79f0b90f7bae6f6f8533240bb60d38c58ec8884caee53ba18aa845R10-R12 as part of release process (structure is TBD, but this will be the place to change)